src/App/_tests_/socialSharing.test.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { wrap } from 'module';
+import { shallow } from '../../enzyme';
+import ArticleSocialSharing from '../Components/ArticleSocialSharing/ArticleSocialSharing';
+
+describe('ArticleSocialSharing', () => {
+  beforeEach(() => {
+    global.FB = { ui: jest.fn() };
+  });
+  const defaultProps = {
+    slug: 'test-slug',
+    title: 'article-title'
+  };
+
+  const renderArticleSocialSharing = (args) => {
+    const final = { ...defaultProps, ...args };
+    return shallow(<ArticleSocialSharing {...final} />);
+  };
+
+  it('renders component', () => {
+    const wrapper = renderArticleSocialSharing();
+    wrapper.find('#twitter-share').simulate('click');
```

Comment:
@murediane is there a way to confirm that each of these clicks works independently of each other? Because I can see 4 clicks but one assertion.

Reply:
there is nothing I can expect because they opening external windows . I am just asserting if these cases are being executed.